nextflow_schema.json

```
@@ -88,27 +93,29 @@
             "properties": {
                 "digestion": {
                     "type": "string",
-                    "default": "hindiii",
-                    "description": "Name of restriction enzyme to automatically set the restriction_site and ligation_site options"
+                    "default": null,
+                    "description": "Name of restriction enzyme to automatically set the restriction_site and ligation_site options (hindiii, mboi, dpnii, arima)"
```

Couldn't this have a set of valid params? 
```suggestion
                    "description": "Name of restriction enzyme to automatically set the restriction_site and ligation_site options (hindiii, mboi, dpnii, arima)",
                    "enum": ["hindiii", "mboi", "dpnii", "arima"],

```